data_steward/cdr_cleaner/cleaning_rules/null_concept_ids_for_numeric_ppi.py

```diff
@@ -0,0 +1,134 @@
+"""
+Nullify concept ids for numeric PPIs from the RDR observation dataset
+
+Original Issues: DC-537, DC-703
+
+The intent is to null concept ids (value_source_concept_id, value_as_concept_id, value_source_value,
+value_as_string) for numeric PPIs from the RDR observation dataset. The changed records should be
+archived in the dataset sandbox.
+"""
+
+# Python imports
+import logging
+
+# Project imports
+from constants.cdr_cleaner import clean_cdr as cdr_consts
+from cdr_cleaner.cleaning_rules.base_cleaning_rule import BaseCleaningRule
+from constants.bq_utils import WRITE_TRUNCATE
+
+LOGGER = logging.getLogger(__name__)
+
+SAVE_TABLE_NAME = "dc_703_obs_changed_rows_saved"
+
+# Query to create tables in sandbox with the rows that will be removed per cleaning rule
+SANDBOX_QUERY = """
+CREATE OR REPLACE TABLE 
+    `{project}.{sandbox_dataset}.{intermediary_table}` AS (
+SELECT *
+FROM 
+    `{project}.{dataset}.observation`
+WHERE
+    questionnaire_response_id IS NOT NULL
+AND 
+    value_as_number IS NOT NULL
+AND 
+    (value_source_concept_id IS NOT NULL OR value_as_concept_id IS NOT NULL))
+"""
```

Comment:
Please update queries using Jinja, per the discussion in the eng sync. Here is an example Lauren sent me:

https://github.com/all-of-us/curation/blob/develop/tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py#L44

Reply:
Should I go ahead and set up my Jinja environment in my module? I know we discussed putting it in a utils file.

Reply:
Just checked with Lauren, she suggests the following, so I would go ahead and set up the Jinja env in your own module for now: 

> good question.  if you want to set up the environment for yourself in your own module, then you could go ahead and write your queries with a template the way you expect.  when we implement the utility, you could then just import the environment and remove the environment you created from your module without needing to change your sql and go through another round of SQL reviews.  so I would probably do that for now.

Reply:
Okay, I will set it up globally for now.